metaserver/db.go

```diff
@@ -0,0 +1,72 @@
+package metaserver
+
+import (
+	"skybin/core"
+)
+
+// Interface containing operations that must be implemented for every database backend.
+type metaDB interface {
```

Comment:
Recommendation: Take pointers for all struct arguments to avoid unnecessary copies going to/from the DB.

Reply:
Agree. Will change.